src/Console/Commands/AbstractGeneratorCommand.php

```diff
@@ -273,6 +274,20 @@ protected function replaceClassName(&$stub, $name)
         return $this;
     }
 
+    /**
+     * Replace the model namespace name.
+     *
+     * @param $stub
+     * @return $this
+     */
+    private function replaceModelNamespace(&$stub) {
+
+        $modelNamespace = $this->getApi()->getModelNamespace();
```

Comment:
If model namespace is `null` here, use `$this->laravel->getNamespace()` to get the application namespace.

Reply:
done.